api/health/worker.go

```diff
@@ -126,9 +127,13 @@ func (w *worker) Results(tags ...string) (map[string]Result, bool) {
 func (w *worker) Start(ctx context.Context, freq time.Duration) {
 	w.startOnce.Do(func() {
 		detachedCtx := utils.Detach(ctx)
+		w.wg.Add(1)
 		go func() {
 			ticker := time.NewTicker(freq)
-			defer ticker.Stop()
+			defer func() {
```

Comment:
yo dawg I heard you like anonymous functions so I put an anonymous function in your anonymous function in an anonymous function so you can func while you func